send/send.c

```diff
@@ -608,6 +617,85 @@ void mutt_make_attribution(struct Mailbox *m, struct Email *e, FILE *fp_out,
   fputc('\n', fp_out);
 }
 
+static const char *greeting_string(char *buf, size_t buflen, size_t col, int cols,
+                                   char op, const char *src, const char *prec,
+                                   const char *if_str, const char *else_str,
+                                   intptr_t data, MuttFormatFlags flags)
```

Comment:
Please can you add a Doxygen function comment.
You can strip down [index_format_str()'s](https://github.com/neomutt/neomutt/blob/master/hdrline.c#L411)

Reply:
> Argh! What an idiot **I am**.
> `index_format_str()` isn't any use, because some of the expandos use the 'from' address not the 'to' address.

No-no-no, in tests everything works as expected.
<img width="960" alt="Снимок экрана 2021-03-12 в 9 40 48" src="https://user-images.githubusercontent.com/74021264/110908287-40d32d00-8317-11eb-914b-83970132f00e.png">

Reply:
It might be logically more correct to leave greeting_string().